tests/compat/README.md

```diff
@@ -0,0 +1,31 @@
+# GreptimeDB compatibility test
+
+The compatibility test check whether a newer version of GreptimeDB can read from the data written by an old version of
+GreptimeDB (the "backward" compatibility), and vice-versa (the "forward" compatibility). It's often ran in the Github
```

Comment:
Is "forward compatibility" necessary?

Reply:
It's necessary. If someone upgrade to higher version, ran for few days, only to find some degression that he decides to downgrade. If forward compat is not provided, he may have trouble doing that.